tests/input_output/input/enum.txt

```diff
@@ -0,0 +1,5 @@
+SHOW ENUMS;
+CREATE (n :l1 {s: Status::Good}) RETURN n;
+CREATE (n :l2 {s: {__type: "test", __value: "test_value"}}) RETURN n;
+CREATE (n :l3 {s: {__type: "mg_enum", __value: "some_value"}}) RETURN n;
```

Comment:
I know this is possible, but its a bad test.
Either we are ok with the current impreciseness of using enum value raw. In that case I think we shouldn't have a test. Or we want to be precise, hence `{__type: "mg_enum", __value: "some_value"}` should not be pretty printed as a literal enum because `some_value` is not enum literal.